<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
	background-color: grey;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add User</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="verticalGroupBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>401</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="userUsername"/>
    </item>
    <item>
     <widget class="QTextEdit" name="userPassword"/>
    </item>
    <item>
     <widget class="QTextEdit" name="userRole"/>
    </item>
    <item>
     <widget class="QTextEdit" name="userImage"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>160</width>
     <height>190</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Franklin Gothic Demi Cond</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Franklin Gothic Demi Cond</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Franklin Gothic Demi Cond</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Franklin Gothic Demi Cond</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
